cou/cli.py

```diff
@@ -72,6 +74,17 @@ def _missing_(cls, value: object) -> Enum:
         raise ValueError(f"{value} is not a valid member of VerbosityLevel.")
 
 
+def prompt_message(parameter: str) -> str:
```

Comment:
I'm a little hesitant of having two function with the same name `prompt_message` here and at `execute.py`. If we can't have a single function to deal with both cases, I think we should name then differently

Reply:
My initial idea was to have them sharing the same function, but was worried that maybe it is a bit mixed in logic. I will upload a patch with that approach and we can discuss how much we like it.